.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,8 @@
 <!-- What's in this pull request? -->
 Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.
 
-<!-- If this pull request resolves any bugs in the Swift bug tracker, provide a link: -->
-Resolves SR-NNNN.
+<!-- If this pull request resolves any GitHub issues, mention them: -->
```

Comment:
```suggestion
<!-- If this pull request resolves any GitHub issues, mention them: -->
<!--
If this pull request resolves any GitHub issues, link them.
For more information about linking a pull request to an issue, see:
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
-->
```

Reply:
Added this link and kept the two issues as an example, let me know if this is okay.